src/wram.asm

```diff
@@ -102,18 +102,18 @@ wPlayerBench5CardStage:: ; c2d3
 	ds $1
 
 ; changed type from Venomoth's Shift Pokemon Power
-; if bit 7 == 1, then bits 0-3 override the Pokemon's actual type
-wPlayerArenaCardChangedType:: ; c2d4
+; if bit 7 == 1, then bits 0-3 override the Pokemon's actual color
+wPlayerArenaCardChangedColor:: ; c2d4
 	ds $1
-wPlayerBench1CardChangedType:: ; c2d5
+wPlayerBench1CardChangedColor:: ; c2d5
 	ds $1
-wPlayerBench2CardChangedType:: ; c2d6
+wPlayerBench2CardChangedColor:: ; c2d6
 	ds $1
-wPlayerBench3CardChangedType:: ; c2d7
+wPlayerBench3CardChangedColor:: ; c2d7
 	ds $1
-wPlayerBench4CardChangedType:: ; c2d8
+wPlayerBench4CardChangedColor:: ; c2d8
 	ds $1
-wPlayerBench5CardChangedType:: ; c2d9
+wPlayerBench5CardChangedColor:: ; c2d9
```

Comment:
Similar to the other one. None of these had to be changed, the canonical name is "Type"